pages/boards/components/Posts/Input/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import styles from "./styles.module.scss";
+import searchIcon from "@/assets/icons/ic_search.svg";
+import Image from "next/image";
+
+interface InputProps {
+  placeholder?: string;
+  onSearch: (searchTerm: string) => void;
+}
+
+export default function Input({ placeholder, onSearch }: InputProps) {
```

Comment:
```suggestion
interface InputProps extends React.InputHTMLAttributes<HTMLInputElement> {
  onSearch: (searchTerm: string) => void;
}

export default function Input({ 
  placeholder = "검색할 키워드를 입력해주세요", 
  onSearch, 
  onChange,
  ...props 
}: InputProps) {

//...
      <input
        className={styles["input"]}
        type="search"
        name="searchItems"
        placeholder={placeholder}
        value={inputValue}
        onChange={onChangeSearch}
        {...props}
      />
```
를 해주면 조금 더 공통컴포넌트로서의 역할을 잘 수행할 수 있을 것 같습니다!